Dockerfile-windowsservercore.template

```diff
@@ -4,6 +4,7 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 
 ENV PYTHON_VERSION %%PLACEHOLDER%%
 ENV PYTHON_RELEASE %%PLACEHOLDER%%
+ENV PYTHONIOENCODING UTF-8
```

Comment:
This should move up after `SHELL` but before `PYTHON_VERSION`, and ought to include a link (similar to what we did in #278):

```suggestion

# https://github.com/docker-library/python/pull/557
ENV PYTHONIOENCODING UTF-8

ENV PYTHON_VERSION %%PLACEHOLDER%%
ENV PYTHON_RELEASE %%PLACEHOLDER%%
```

Reply:
Thanks, I modified it.